spec/features/creating_peeps_spec.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../web_helper'
+
+feature 'User posts a new message' do
+  let(:email) { 'maker@makers.com' }
+  let(:password) { '000000' }
+  let!(:user) do
+    User.create(email: email, password: password)
+  end
+
+  scenario 'I can post a message to chitter' do
+    sign_in(email: email, password: password)
+    body = 'Message 1'
+    post_peep(body: body)
+    expect(page).to have_content(body)
+  end
+
+  scenario 'I can see when my peep was posted' do
+    sign_in(email: email, password: password)
+    body = 'Message 1'
+    time = Time.new.strftime("%b %-d %H:%M")
+    post_peep(body: body)
+    expect(page).to have_content(time)
+  end
+
+  scenario 'I can see a list with my peeps' do
+    sign_in(email: email, password: password)
+    body1 = 'Message 1'
+    post_peep(body: body1)
+    body2 = 'Message 2'
+    post_peep(body: body2)
+    expect(page).to have_content(body1)
+    expect(page).to have_content(body2)
+  end
+
+  scenario 'I can see all peeps in reverse chronological order' do
+    sign_in(email: email, password: password)
+    body1 = 'Message 1'
+    post_peep(body: body1)
+    body2 = 'Message 2'
+    post_peep(body: body2)
+    expect(page).to have_selector('ul li:nth-child(1)', text: body2)
```

Comment:
To get this test working, try manually setting up the conditions (by actually creating the peeps in the browser) and then using `document.querySelector("ul li:nth-child(1)")` in the browser dev console to see if the element you expect gets selected.  (If I'm not being clear, just grab me and we can talk it through!)